### ISSUE TITLE
Json API live data url

### ISSUE THREAD

**Issue by IVEN2022** [manifestinteractive/teleprompter#50]
Hello and thanks for this amazing autocue , is there a way that it can read feed from json api url ?

**Comment by manifestinteractive** [comment]
@IVEN2022 can you explain a little bit more what you are asking for?  Ideally, I'd like to know how you intend to use this, as a step by step process where you walk me through how you envision this feature request?

**Comment by IVEN2022** [comment]
Hello and Happy New Year , regarding the json api  , if it possible to read from json api  get request

this is for example the json api get request raw data i am getting from an ip

https://pastebin.com/raw/f6MT0Qb4

**Comment by IVEN2022** [comment]
I hope this is not abundant yet

**Comment by manifestinteractive** [comment]
@IVEN2022 If you can, it would be helpful for me to have you walk me through how you would expect this to work.

Main things that seem like they'd be issues to address:

1. Where would you expect to enter a URL on this app? Are you looking to have a button or something added to enter a URL?
2. Once we get a response from JSON output, how do you envision this that output getting rendered? In the example you provided, that was an array with two items, and since JSON output can literally be anything, as a user, you'd need some way to select what you want to show ... so, what does that look like for you?

Basically, I just need you to walk me through what you want out of this feature, starting from what we have now, and walk me through how the steps you imagine you would need to solve this for you.  Without that info, I cannot really help, and would need to close this feature request out :(

**Comment by manifestinteractive** [comment]
Closing this one out since the requested feedback was not provided.
